### ISSUE TITLE
License information?

### ISSUE THREAD

**Issue by mkdolan** [dhrp/docker-tutorial#3]
It appears there is no license information associated with docker-tutorial...

**Comment by dhrp** [comment]
Yeah, I'll add one soon. If you're interested in using it, know that I'm working on a full rewrite which will make it much easier to do so. 

I'm thinking to use the MIT licence. Would that work for you?

Best,
Thatcher

;-) - mobiele email

**Comment by mkdolan** [comment]
Sure would work. When is the rewrite coming? Was just looking into playing with it for some ideas I had and wondered what license it was under. This is the most awesome demo tool I've ever seen.

:-)

**Comment by joshisa** [comment]
@dhrp Looked through the repo and didn't see any inclusion of your proposed MIT license.  We're thinking about using/modifying this to help doc some CLI , but need a license listed in the original repo.  Apache 2.0 or MIT works for us if that's still your inclination.  Thanks.
